cfg/defaultConstants.js

```diff
@@ -1,27 +1,41 @@
 module.exports = {
 
-    baseUrl: JSON.stringify("http://localhost:50000/"),
+  baseUrl: 'http://localhost:50000/',
 
-    authRedirectUri: JSON.stringify("http://localhost:3000/"),
+  authRedirectUri: 'http://localhost:3000/',
 
-    retries: 2,
+  retries: 2,
 
-    retryExponentialBackoffFactor: 2,
+  retryExponentialBackoffFactor: 2,
 
-    retryMinTimeoutInMiliseconds: 50,
+  retryMinTimeoutInMiliseconds: 50,
 
-    retryMaxTimeoutInMiliseconds: 1500,
+  retryMaxTimeoutInMiliseconds: 1500,
 
-    ticketRefreshIntervalInSeconds: 300,
+  ticketRefreshIntervalInSeconds: 300,
 
-    authVersion: "'ADAL'",
+  authVersion: 'ADAL',
 
-//   Leave this as is if your code is in a public repo (or delete it if you want).  You can override the defaults with the real information in your cfg/$env.const.js file.  
+  //   Leave this as is if your code is in a public repo (or delete it if you want).
+  // You can override the defaults with the real information in your cfg/$env.const.js file.
+  aadInstance: 'https://login.microsoftonline.com/', // Replace only if using AAD non-public instances.
 
-    aadInstance: "'https://login.microsoftonline.com/'", //Replace only if using AAD non-public instances. Yes, use both single and double quotes.
+  aadTenant: 'Your Azure Active Directory tenant here',
 
-    aadTenant: "'Your Azure Active Directory tenant here'", //Yes, use both single and double quotes.
-
-    clientId: "'Your EventUI Azure Active Directory app registration client ID here'" //Yes, use both single and double quotes.
+  clientId: 'Your EventUI Azure Active Directory app registration client ID here',
 
+  ticketSystems: {
```

Comment:
Couldn't you use a template literal since this is a JS file?

Reply:
I considered doing that, but since we don't have the ticketId variable in that scope this would require to eval() it at run time, or do some sort of string replacement, or a similar approach. 

Another approach would be to convert it into (ticketId) => `http://example1/ticket/${ticketId}/something`, but this will be serialized as JSON anyways, so it'll require some deserialization on the other end. 

At the end, having prefix/suffix instead seemed liked a good simple compromise.

Reply:
FWIW - JavaScript people love them some Evals - so it's not a weird thing to do at all in the language. That said, I'm okay with your choice.